vsintegration/ProjectTemplates/TutorialProject/Template/Tutorial.fsx

```diff
@@ -45,108 +64,146 @@ module IntegersAndNumbers =
     let sampleFunction2 (x:int) = 2*x*x - x/5 + 3
 
     let result2 = sampleFunction2 (7 + 4)
     printfn "The result of applying the 1st sample function to (7 + 4) is %d" result2
 
     /// Conditionals use if/then/elid/elif/else.
     ///
-    /// Note that F# uses whitespace indentation-aware syntax like Python.
+    /// Note that F# uses whitespace indentation-aware syntax, similar to languages like Python or Ruby.
     let sampleFunction3 x = 
         if x < 100.0 then 
             2.0*x*x - x/5.0 + 3.0
         else 
             2.0*x*x + x/5.0 - 37.0
 
     let result3 = sampleFunction3 (6.5 + 4.5)
-    printfn "The result of applying the 2nd sample function to (6.5 + 4.5) is %f" result3
 
+    // This line uses '%f' to print the result as a float.  As with '%d' above, this is type-safe.
+    printfn "The result of applying the 2nd sample function to (6.5 + 4.5) is %f" result3
 
 
-/// Strings and booleans are fundamental data types in F# programming.
-module StringManipulation = 
+/// Booleans are fundamental data types in F#.  Here are some examples of Booleans and conditional logic.
+///
+/// To learn more, see: 
+///     https://docs.microsoft.com/en-us/dotnet/articles/fsharp/language-reference/primitive-types
+///     and
+///     https://docs.microsoft.com/en-us/dotnet/articles/fsharp/language-reference/symbol-and-operator-reference/boolean-operators
+module Booleans =
 
-    /// Booleans use 'true', 'false', 'not', '&&' and '||'
+    /// Booleans values are 'true' and 'false'.
     let boolean1 = true
     let boolean2 = false
 
+    /// Operators on booleans are 'not', '&&' and '||'.
     let boolean3 = not boolean1 && (boolean2 || false)
 
-    printfn "The expression 'not boolean1 && (boolean2 || false)' is %A" boolean3
+    // This line uses '%b'to print a boolean value.  This is type-safe.
+    printfn "The expression 'not boolean1 && (boolean2 || false)' is %b" boolean3
+
+
+/// Strings are fundamental data types in F#.  Here are some examples of Strings and basic String manipulation.
+///
+/// To learn more, see: https://docs.microsoft.com/en-us/dotnet/articles/fsharp/language-reference/strings
+module StringManipulation = 
 
     /// Strings use double quotes.
     let string1 = "Hello"
     let string2  = "world"
 
-    /// Strings can also use @ to create a verbatim string literal
-    let string3 = @"c:\Program Files\"
+    /// Strings can also use @ to create a verbatim string literal.
+    /// This will ignore escape characters such as '\', '\n', '\t', etc.
+    let string3 = @"C:\Program Files\"
 
-    /// String literals can also use triple-quotes
-    let string4 = """He said "hello world" after you did"""
+    /// String literals can also use triple-quotes.
+    let string4 = """The computer said "hello world" when I told it to!"""
 
-    /// String concatenation is '+'. Also see String.concat, System.String.Join and others.
+    /// String concatenation is normally done with the '+' operator.
```

Comment:
normally seems fine here.  There are after all other ways of concatenating strings..